pages/linux/shiny-mirrors.md

```diff
@@ -0,0 +1,20 @@
+# shiny-mirrors
+
+> Generate a pacman mirror list for Manjaro Linux.
+> Every run of shiny-mirrors requires you to synchronize your database and update your system using `sudo pacman -Syyu`.
```

Comment:
If the Manjaro Wiki or any another doc page is not useful for this command use the Git repository:
```suggestion
> Every run of shiny-mirrors requires you to synchronize your database and update your system using `sudo pacman -Syyu`.
> More information: <https://gitlab.com/Arisa_Snowbell/shiny-mirrors>.
```

Reply:
There is this manual page basically 
https://gitlab.com/Arisa_Snowbell/shiny-mirrors/-/blob/domina/shiny-mirrors/man/shiny-mirrors.md
Would that be better maybe?

Reply:
Link the MD file then.

Reply:
Oh, it did not update in my browser so I made the commit myself without this commit suggestion feature
I don't use github much